### DISCUSSION TITLE
Internal server error when using the form component

### DISCUSSION THREAD

**Discussion by brucegl** [huntabyte/shadcn-svelte#1138] [Help]
I followed the [form usage section](https://www.shadcn-svelte.com/docs/components/form) but I got this error:
```
[vite] Internal server error: Error while preprocessing /home/gary/Documents/Dev/tapestry/src/lib/components/ui/form/form-element-field.svelte - Transform failed with 1 error:
/home/me/Documents/Dev/project/src/lib/components/ui/form/form-element-field.svelte:1:0: ERROR: Unexpected ","
  Plugin: vite-plugin-svelte
  File: /home/me/Documents/Dev/project/src/lib/components/ui/form/form-element-field.svelte
   
   Unexpected ","
   1  |  , U extends FormPathLeaves<T>">
      |  ^
   2  |         import type { HTMLAttributes } from "svelte/elements";
   3  |         import * as FormPrimitive from "formsnap";
```
I took a look at form-element-field.svelte and on line 7, `<script lang="ts" generics="T extends Record<string, unknown>, U extends FormPathLeaves<T>">`

this error is given in VScode:
```
Transform failed with 1 error:
/home/gary/Documents/Dev/tapestry/src/lib/components/ui/form/form-element-field.svelte:1:0: ERROR: Unexpected ","svelte(script)
```
Not sure what the issue is. I'll create a reproduction if needed.

**UPDATE...** 
I created a reproduction on [stackblitz](https://stackblitz.com/edit/github-dfkaut?file=src%2Froutes%2Fschema.ts) that works, however, in form-element-field.svelte, the closing chevron on the same  `<script lang="ts" generics="T extends Record<string, unknown>, U extends FormPathLeaves<T>">` line is showing red, so something doesn't look quite right.

Thanks

**Comment by brucegl** [comment]
Hmm, no error on a fresh project.... I think it could be npm/vite related. I've tried upgrading these to no avail.

I've yet to deploy my project live, so I'm tempted to do a complete upgrade (including SK2 and SV5), but an easy fix would be very welcome at this point in time.

Holding off for the cavalry!

**Reply by griessel-jc** [reply]
Did you manage?

**Reply by brucegl** [reply]
I created a new project and moved all my code to it
